src/50_Samples_%26_Templates/Product_Page.html

```diff
@@ -629,15 +637,14 @@ <h2 class="m1">Apple</h2>
     selected
     option="a">ABOUT</div>
   <div role="tabpanel"
-      class="tabContent p1 p">Fruit are righ of vitamins.</div>
+      class="tabContent p1 p">Fruit are rich of vitamins.</div>
   <div role="tab"
     class="tabButton h4"
     option="b">SPECS</div>
   <div role="tabpanel"
-    class="tabContent p1 p">Always wash fruits.</div>
+    class="tabContent p1 p">Always wash fruits before eating.</div>
```

Comment:
Grammar edit + lorem ipsum:

"Always wash fruit before eating. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."